firewalld.spec

```diff
@@ -119,15 +121,24 @@ firewalld.
 
 %prep
 %autosetup
+%if 0%{?rhel} < 8
```

Comment:
This change is not necessary as of 3f7419dcb50a5bdcf800512d24919c33a413c71b. We only rely on python3-nftables and no longer check for the `nft` binary.